### DISCUSSION TITLE
Hot engine :-p ....

### DISCUSSION THREAD

**Discussion by CptBayer** [flybywiresim/aircraft#4744] [General]
Hi developers,

I think a known issue. The engine is quite hot after cold and dark status and before starting them for the first legof the day :-p...

![grafik](https://user-images.githubusercontent.com/76531893/117536827-a123d980-affd-11eb-8bbd-e5d4f7d4cf88.png)

Greets Klaus

**Comment by hotshotp** [comment]
yes we have many times stated on the discord and other places that the engine model is a work in progress
